doc/decisions/database_plant_hierarchy.md

```diff
@@ -34,6 +34,8 @@ See the [PSQL documentation](https://www.postgresql.org/docs/current/ddl-inherit
 
 > Table inheritance is typically established when the child table is created, using the INHERITS clause of the CREATE TABLE statement.
 
+> Rust Diesel isn't intended for that. To only select data from a specific table, and not include all child tables, we would need to use the `FROM ONLY` keyword, which is not implemented in Rust Diesel.
```

Comment:
```suggestion
> Rust Diesel isn't intended for that.
> To only select data from a specific table, and not include all child tables, we would need to use the `FROM ONLY` keyword, which is not implemented in Rust Diesel.
```

Sentences need to be written on separate lines according to our documentation guidelines.